source/elements/oneTBB/source/task_scheduler/scheduling_controls/resumable_tasks.rst

```diff
@@ -23,6 +23,7 @@ Requirements:
 
 The ``oneapi::tbb::task::suspend`` function called within a running task suspends execution of the task and switches the thread to participate in other oneTBB parallel work.
 This function accepts a user callable object with the current execution context ``oneapi::tbb::task::suspend_point`` as an argument.
+The user-specified callable object is executed by the calling thread.
```

Comment:
Do we have the info somewhere about parameters a user can specify it with?

Reply:
@alexandraepan https://github.com/oneapi-src/oneAPI-spec/blob/main/source/elements/oneTBB/source/named_requirements/task_scheduler/suspend_func.rst